spec/requests/rack_attack_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe 'throttling requests', allowed_extra_analytics: [:*] do
+RSpec.describe 'throttling requests' do
```

Comment:
@aduth is that the correct way to get this in compliance? 
This bubbled up when I rebased
https://gitlab.login.gov/lg/identity-idp/-/jobs/1197662

Reply:
I think this might be an issue on main, possibly from someone else's changes.